app/services/ncr/approval_manager.rb

```diff
@@ -1,4 +1,6 @@
 module Ncr
+  START_OF_NEW_6X_APPROVAL_POLICY = Time.zone.local(2016, 7, 5, 0, 0, 0)
```

Comment:
Can this just be a feature flag to check if it's on? This will give us flexibility on launch date (which could be turned on July 1st) and allow us to easily revert back in case the policy is reverted.